cloud-claim-check-pattern/README.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: pattern
+title: Claim Check Pattern
+folder: Cloud-Claim-Check-Pattern
+permalink: /patterns/cloud-claim-check-pattern/
+categories: Cloud
+tags:
+  - Cloud distributed
+  - Microservices
+---
+
+## Name
+
+[Claim Check Pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/claim-check)
+
+## Also known as
+
+[Reference-Based Messaging](https://www.enterpriseintegrationpatterns.com/patterns/messaging/StoreInLibrary.html)
+
+## Intent
+
+- Reduce a load of data transfer through the Internet - Instead of sending actual data directly, just send the message reference. (location details where the message is stored.)
```

Comment:
```suggestion
- Reduce the load of data transfer through the Internet. Instead of sending actual data directly, just send the message reference.
```